app/Controllers/authController.php

```diff
@@ -205,7 +205,7 @@ public function registerAction() {
 			Minz_Error::error(403);
 		}
 
-		$this->view->show_tos_checkbox = file_exists(DATA_PATH . '/tos.html');
+		$this->view->show_tos_checkbox = file_exists(join_path(DATA_PATH, 'tos.html'));
```

Comment:
I am not sure this adds any benefit, does it? (and it is slower, longer, and custom)
No need to change it back, but just for future cases. Windows does not need a special case, at least.

Reply:
Oh, I didn't know that! I take note for the next time :)